migrations/schemas/20220620225736-guild_config_sales_trackers.sql

```diff
@@ -0,0 +1,12 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS guild_config_sales_trackers (
```

Comment:
you already create `guild_config_sales_tracker` if need `guild_config_sales_trackers`
then this migration needs to be 
```
ALTER TABLE guild_config_sales_tracker RENAME TO guild_config_sales_trackers
```